modules/sr_site/manifests/init.pp

```diff
@@ -86,6 +86,11 @@
   include sr_site::login
   include sr_site::meta
 
+  class { 'sr_site::discord_gated_entry':
+    git_root => $git_root,
+    root_dir => '/srv/discord-gated-entry',
+  }
+
```

Comment:
Should we _always_ install this? Or perhaps only when `$competitor_services or $volunteer_services` or maybe only when `$volunteer_services`?

Reply:
Perhaps we only want this when `not $volunteer_services`? I think for now yes, but we'll want to review its installation for next year.

Reply:
There are three service modes, I think we should work on the basis of inclusion rather than exclusion.

Reply:
d231499